src/scripts/main.js

```diff
@@ -1,3 +1,259 @@
+
+employee.classList.add('new-employee-form');
+document.body.appendChild(employee);
+
+// #region creating a notification
+function notificationMessage(text, type) {
+  const notification = document.createElement('div');
+  const titleNotification = document.createElement('h3');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.classList.add('notification');
+  titleNotification.classList.add('title');
+
+  notification.appendChild(titleNotification);
+
+  notification.classList.add(type);
+  titleNotification.textContent = text;
+
+  document.body.appendChild(notification);
+
+  setTimeout(() => {
+    document.body.querySelector('.notification').remove();
+  }, 10000);
+}
+// #endregion
+
+// #region salary formatting
+function formatCurrency(numberStr) {
+  const number = parseInt(numberStr, 10);
+
+  return '$' + number.toLocaleString('en-US');
+}
+// #endregion
+
+// #region creating a form (constructor)
+function newForm(atributes, cities) {
+  const form = document.createElement('form');
+
+  for (const [key, value] of Object.entries(atributes)) {
+    const label = document.createElement('label');
+
+    if (value === 'Office') {
+      const select = document.createElement('select');
+
+      label.textContent = `${value}:`;
+      select.setAttribute('data-qa', `${key}`);
+      select.name = key;
+
+      cities.forEach((city) => {
+        const option = document.createElement('option');
+
+        option.value = city;
+        option.textContent = city;
+
+        select.appendChild(option);
+      });
+
+      label.appendChild(select);
+      form.appendChild(label);
+    } else {
+      const input = document.createElement('input');
+
+      label.textContent = `${value}:`;
+      input.setAttribute('data-qa', `${key}`);
+      input.name = `${key}`;
+      input.type = 'text';
+      input.setAttribute('required', '');
+
+      label.appendChild(input);
+      form.appendChild(label);
+    }
+  }
+
+  const button = document.createElement('button');
+
+  button.textContent = 'Save to table';
+  button.type = 'submit';
+
+  form.appendChild(button);
+
+  return form;
+}
+
+// #endregion
+
+// #region form submit event
+employee.querySelector('button').addEventListener('click', (e) => {
+  const inputName = employee.querySelector('[data-qa="name"]');
+  const inputPosition = employee.querySelector('[data-qa="position"]');
+  const inputOffice = employee.querySelector('[data-qa="office"]');
+  const inputAge = employee.querySelector('[data-qa="age"]');
+  const inputSalary = employee.querySelector('[data-qa="salary"]');
+
+  const valuesOfInputs = [
+    inputName,
+    inputPosition,
+    inputOffice,
+    inputAge,
+    inputSalary,
+  ];
+
+  if (
+    inputName.value.trim() !== '' &&
+    inputName.value.length >= 4 &&
+    inputAge.value.trim() !== '' &&
+    parseInt(inputAge.value) >= 18 &&
+    parseInt(inputAge.value) <= 90 &&
+    inputPosition.value !== '' &&
+    inputSalary.value !== ''
```

Comment:
So now you check it twice - here, inside the if condition, and below to show the error message. Instead, validate all values in a separate function. If some value is invalid show error message (only one at the time to prevent overlapping).

Return true/false from the function and use this value here